docs/proj1rubric.md

```diff
@@ -1,45 +1,87 @@
+13| Use of  style checkers |3| in GH|
+14| Use of code  formatters. |3| in GH|
+15| Use of syntax checkers. |3|in GH|
+16| Use of code coverage |0||
+17| other automated analysis tools|2|in GH|
+18| test cases exist|3| https://github.com/shahrk/feature-hunt/tree/main/test|
+19| test cases are routinely executed|3|in GH|
+20| the files CONTRIBUTING.md lists coding standards and lots of tips on how to extend the system without screwing things up|3|https://github.com/shahrk/feature-hunt/blob/main/CONTRIBUTING.md|
+21| issues are discussed before they are closed|3|every issue is dicussed by all, then assigned to one appropriate person for closure|
+22| Chat channel: exists|3| Google chat was used for communication |
+23| test cases:.a large proportion of the issues related to handling failing cases.|3|in GH|
+24| evidence that the whole team is using the same tools: everyone can get to all tools and files|3||
+25| evidence that the whole team is using the same tools (e.g. config files in the repo, updated by lots of different people)|3|https://github.com/shahrk/feature-hunt/blob/ed52d3273606ef96869c77584c5c5ebdd455810c/backend/requirements.txt highlights that the same libraries are used by all|
+26| evidence that the whole team is using the same tools (e.g. tutor can ask anyone to share screen, they demonstrate the system running on their computer)|3||
+27| evidence that the members of the team are working across multiple places in the code base|3| https://github.com/shahrk/feature-hunt/graphs/contributors|
+28|short release cycles | 3||
+29|Documentation provide a clear, high-level overview of your software|3| https://github.com/shahrk/feature-hunt/blob/main/README.md |
+30|Documentation clearly describes the type of user who should use your software|3| Told in the video https://github.com/shahrk/feature-hunt/blob/main/README.md|
+31|case studies to show how your software has been used by yourself and others|3|Screenshots in https://github.com/shahrk/feature-hunt/blob/main/README.md|
+32|Documentation includes a "quick start" guide that provides a short overview of how to use your software with some basic examples of use.|3|https://github.com/shahrk/feature-hunt/blob/main/README.md|
+33|Documentation provides clear, step-by-step instructions on how to deploy and use your software.|3|https://github.com/shahrk/feature-hunt/blob/main/README.md|
+34|A comprehensive guide to all your software’s commands, functions and options.|3|https://github.com/shahrk/feature-hunt/blob/main/README.md|
+35|If your software can be used as a library, package or service by other software, do you provide comprehensive API documentation?|3|Yes, Full documentation is present in GH readme|
+36|Troubleshooting information that describes the symptoms and solutions for problems.|1|Kept as future scope|
+37|Documentation stored under revision control with your source code.|1|Its considered as Future Scope|
```

Comment:
Documentation is on GitHub so it is by default stored under revision control

Reply:
Modified in the new commit